src/ruby/install.sh

```diff
@@ -177,9 +177,11 @@ fi
 find_version_from_git_tags RUBY_VERSION "https://github.com/ruby/ruby" "tags/v" "_"
 
 # Just install Ruby if RVM already installed
-if [ $(rvm --version) != "" ]; then
+if rvm --version > /dev/null; then
     echo "Ruby Version Manager already exists."
-    if [ "${RUBY_VERSION}" != "none" ]; then
+    if [[ "$(ruby -v)" = *"${RUBY_VERSION}"* ]]; then
```

Comment:
At this point I believe `RUBY_VERSION` is a fully expanded version (from line 177), right?

Reply:
Yes, that's correct.

![image](https://github.com/devcontainers/features/assets/24955913/93c39a3c-9cb5-4df0-9ccb-6980b26af1de)